examples/list-collection.py

```diff
@@ -1,3 +1,8 @@
-from pestuary.collections import collection_list, collection_list_content
-response = collection_list()
-print(response)
+from pestuary.collections import collection_list
+
+def example_collection_list():
+    response = collection_list()
+    print(response)
+
+if __name__ == '__main__':
+    example_collection_list()
```

Comment:
Same as above. Putting this code in a function does not make it more readable and it confuses the example